pkgs/os-specific/linux/kernel/common-config.nix

```diff
@@ -85,6 +101,31 @@ let
       # depends on HAVE_VIRT_CPU_ACCOUNTING_GEN depends on 64BIT,
       # so we can't force-enable this
       RCU_LAZY                         = whenAtLeast "6.2" (option yes);
+
+      # Auto suspend Bluetooth devices at idle
+      BT_HCIBTUSB_AUTOSUSPEND          = yes;
+
+      # Expose cpufreq stats in sysfs
+      CPU_FREQ_STAT                    = yes;
+
+      # Enable CPU energy model for scheduling
+      ENERGY_MODEL                     = whenAtLeast "5.0" yes;
+
+      # Enable scheduling stats collection
+      SCHEDSTATS                       = yes;
+
+      # Enable thermal interface netlink API
+      THERMAL_NETLINK                  = whenAtLeast "5.9" yes;
+
+      # Prefer power-efficient workqueue implementation to per-CPU workqueues,
+      # which is slightly slower, but improves battery life.
+      # This can be disabled at runtime to get performance back,
```

Comment:
s/runtime/boot time/? "runtime" to me implies that you can adjust this using sysfs while the system is up, which doesn't seem to work.

Reply:
If I'm reading the docs correctly, it should? But I can change the comment if it doesn't.

Reply:
After changing it to yes using the kernel parameter on my current system:
```
aloisw@genesis ~> echo n | doas tee /sys/module/workqueue/parameters/power_efficient
tee: /sys/module/workqueue/parameters/power_efficient: Permission denied
n
aloisw@genesis ~ [0|1]> cat /sys/module/workqueue/parameters/power_efficient
Y
```

Reply:
You can't change the kernel module option, but each individual workqueue can opt into being power efficient: https://docs.kernel.org/core-api/workqueue.html#flags

Reply:
How does this relate to disabling power-efficient workqueue implementation at runtime? Can I somehow change that flag at runtime? My understanding is that these flags are set by the kernel code.

Reply:
Updated the comment to be clearer about this.